auto_nag/bzcleaner.py

```diff
@@ -509,15 +509,15 @@ def send_email(self, date='today', dryrun=False):
                     body,
                     html=True,
                     login=login_info,
-                    dryrun=dryrun,
+                    dryrun=self.dryrun,
                 )
             except:  # NOQA
                 logger.exception('Tool {}'.format(self.name()))
                 status = 'Failure'
 
             db.Email.add(self.name(), receivers, 'global', status)
             if isinstance(self, Nag):
-                self.send_mails(title, dryrun=dryrun)
+                self.send_mails(title)
```

Comment:
you must update send_mails in nag_me.py

Reply:
Added this back in 4afa166 so I don't have to touch nag_me.py